File: tacvi/an_ukun_lifebleeder.pl
sub EVENT_DEATH {
$Check_EntID = $entity_list->GetMobByNpcTypeID(298139);
if ($Check_EntID) {
quest::emote("flesh and bones are reformed by dark magic");
quest::spawn2(298143,0,0, $x, $y, $z, 0.5);
}
}

sub EVENT_SIGNAL {

if ($signal == 1) {
$Death = 1;
}
}

File: WorkInProgress/tacvi/an_ukun_lifebleeder.lua
-- Add during PXK fight

function event_spawn(e)
	e.self:Emote("flesh and bone are reformed by dark magic");
end

function event_death_complete(e)
	-- On Death, signal the controller that we have died so the controller can keep track of the number of spawns.
	eq.signal(298222, 298043, 0);
end
